superset-frontend/src/components/ListView/Filters/DateRange.tsx

```diff
@@ -62,7 +62,7 @@ function DateRangeFilter(
 
   return (
     <RangeFilterContainer>
-      <FormLabel>{Header}</FormLabel>
+      <FormLabel>{t(Header as string)}</FormLabel>
```

Comment:
Can you explain the "as string" need here? If this is a dynamic value, it might not be wise to use the t() here.